src/js/shared_controls.js

```diff
@@ -620,7 +620,6 @@ function createPokemon(pokeInfo) {
 function getMoveDetails(moveInfo, ability, item, isMax) {
 	var moveName = moveInfo.find("select.move-selector").val();
 	var isZMove = gen === 7 && moveInfo.find("input.move-z").prop("checked");
-	var isMax = gen === 8 && isMax;
```

Comment:
I'm surprised there's no linter to catch variable redeclarations like this...

Reply:
We have eslint, don't we? Did we forget to turn on redeclaration checks?

Reply:
I believe thats a question for @scheibo

Reply:
I mean, I was the one who turned on eslint originally, but I guess he's messed with it since then.

Reply:
Did I? I don't think I messed with eslint at all, it should be the same as however you set it up (though it no longer applies to the code i moved to TS which uses tslint)

Reply:
> `"no-redeclare": "off",`

I REMEMBER THIS NOW

The codebase redeclares variables everywhere; I wanted to turn this on but I couldn't unless all the existing redeclares were fixed.